docs/automated_testing.md

```diff
@@ -991,38 +993,162 @@ uses: suffolkLITLab/ALKiln@releases/v4
+<!-- TODO: Rearrange - Maybe use this section to refer to another section that is more focused on setting arbitrary environment variables in general -->
+
+You can use GitHub secrets to set environment variable values with sensitive information. For example, a password. The value of this variable will not appear anywhere in the report or in the console. You also will be unable to take a screenshot of the page and ALKiln will avoid taking an error screenshot.
+
+:::info
+This is also useful if an organization wants to create a variable that all of its repositories will be able to use.
+:::
 
-1. Follow the [GitHub instructions](https://docs.github.com/en/actions/security-guides/encrypted-secrets) to set one or more GitHub secrets. You can add these to one repository secrets or you can add these to your organization secrets, whichever is right for you.
+1. Follow the [GitHub instructions](https://docs.github.com/en/actions/security-guides/encrypted-secrets) to set one or more GitHub secrets. You can add these to one repository's secrets or you can add these to your organization's secrets, whichever is right for you.
 
 2. Go to the home page of your repository. Tap on the `.github` folder, then on `workflows` folder, then on the YAML file in there that runs the ALKiln tests.
 
-It should include a line that looks like this:
+3. It should include a line that looks like this:
+
 ```yml
-      - uses: actions/checkout@v2
+jobs:
 ```
 
-3. If this is the first environment variable you're adding, add these lines:
+If this is the first environment variable you're adding, right above that line add a new line like this:
 
 ```yml
-      - name: Set env vars
-        run: |
+env:
 ```
 
-4. Whenever you want to add a secret, add a new line under those in this format:
+4. Whenever you want to add a secret, add a new line under `env:` indented once as shown below:
 
 ```yml
-          echo "USER_PASSWORD=${{ secrets.USER_PASSWORD }}" >> $GITHUB_ENV
+  USER_PASSWORD: "${{ secrets.USER_PASSWORD }}"
 ```
 
 `USER_PASSWORD` is just a placeholder in our example. You can name your secrets whatever you want to. Make sure you use the same words as the GitHub secret you made.
 
 5. Write your Step and use the names of these secrets as the values.
 
+If you're not worried about keeping the information secure, you don't have to use a GitHub secret - you can just put the value straight into your workflow file like this:
+
+```yml
+  MAX_SECONDS_FOR_SERVER_RELOAD: "300"
+```
+
+ALKiln will still avoid printing this value and avoid taking screenshots when it is used. There is no way ALKiln can tell which environment variables have sensitive information and which ones are safe.
+
+All together, this section can look similar to this:
+
+```yml
+env:
+  USER_PASSWORD: "${{ secrets.USER_PASSWORD }}"
+  MAX_SECONDS_FOR_SERVER_RELOAD: "300"
+
+jobs:
+```
+
+
+## Your workflow file
+
+This whole section is very technical. Feel free to ask us questions.
```

Comment:
I'd remove this line. The top level section isn't actually very technical, and could help people understand what this file is, and I'm afraid it might scare people off.

Moving it to the end of the top level section (before `Required inputs`) would be useful.